src/leap/bitmask/services/mail/conductor.py

```diff
@@ -84,11 +81,11 @@ def _handle_imap_events(self, req):
         :param req: Request type
         :type req: leap.common.events.events_pb2.SignalRequest
```

Comment:
Thanks, I didn't realize of that change.
I've changed all the events related to this parameter change.
Please take a look to make sure I'm not missing anything.